.github/workflows/pr-if-new-kernel.yml

```diff
@@ -28,8 +28,12 @@ jobs:
             git config --global user.email "gardenlinux@users.noreply.github.com"
             git commit -am 'Update kernel ${{ steps.update.outputs.has-update }}'
             git push --set-upstream origin update-kernel
-            gh pr create --base main --head update-kernel --title 'Update kernel ${{ steps.update.outputs.has-update }}' --body "automated update" --reviewer fwilhe
+            UPDATE_PR_LINK=$(gh pr create --base main --head update-kernel --title 'Update kernel ${{ steps.update.outputs.has-update }}' --body "automated update" --reviewer fwilhe)
```

Comment:
I would put the reviewer into an environment variable. This way, we can change this parameter from outside, without making every time a commit. This might require a bit of code to validate that we have correct usernames.

Reply:
Yeah ideally I would have wanted to use the maintainers group but this did not work for some reason and so I decided to make it work first.. we should investigate how to properly use groups here before fiddling with env vars imo.